ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/health/ReadinessService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.ethereum.jsonrpc.health;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ReadinessService extends HealthService {
+  private static final Logger LOG = LogManager.getLogger();
+
+  private static final int READY_STATUS_CODE = 200;
```

Comment:
Codes seems to be HTTP codes. Could it be HttpResponseStatus instead of setting you own int directly ?
```suggestion
  private static final int READY_STATUS_CODE = HttpResponseStatus.OK.code();
```
or even store the READY_STATUS_CODE as a HttpResponseStatus type.
Also why not having common response codes (like SUCCESS_STATUS_CODE and FAILURE_STATUS_CODE thaat could apply to both readiness and liveness) handled at the parent abstract class level instead of somewhat duplicating them?